frontend/src/pages/home/Home.tsx

```diff
@@ -134,13 +124,13 @@ export default function Home(props: { updateTheme: (theme: "system" | "dark" | "
 			</section>
 			
 			<section className={ "page" } id={ "features" }>
-				<InView as={ "div" } onChange={ inViewOnChange }>
+				<div>
 					<h2 className={ "page-title fade-in" }>Features</h2>
 					<h1 className={ "page-subtitle fade-in" }>What We Offer</h1>
 					<p className={ "page-summary fade-in" }>
 						We offer a wide range of features to help you with your daily tasks.
```

Comment:
i18n

Reply:
As Home is not in our requirements at all (and is undergoing many changes anyway), I will not add translation (yet).